src/sentry/static/sentry/app/views/projects/projectContext.jsx

```diff
@@ -172,11 +177,9 @@ const ProjectContext = createReactClass({
       });
 
       this.api.request(this.getEnvironmentListEndpoint(), {
-        success: loadEnvironments,
-      });
-
-      this.setState({
-        loading: false,
```

Comment:
Yes, since it's already set in the success handler for the project details request, I think we shouldn't set `loading:false` again here before that returns